tex/pgfcirc.defines.tex

```diff
@@ -696,6 +696,24 @@
 \ctikzset{tripoles/european nor port/height/.initial=.65}
 \ctikzset{tripoles/european nor port/reserved/.initial=.6}
 \ctikzset{tripoles/european nor port/input height/.initial=.6}
+%
+% switches for op amps
+% changing input polarity
+%
+\newif\ifpgf@circuit@oa@iplusup\pgf@circuit@oa@iplusupfalse
+\pgfkeys{/tikz/noinv input up/.add code={}{\pgf@circuit@oa@iplusuptrue}}
+\ctikzset{noinv input up/.add code={}{\pgf@circuit@oa@iplusuptrue}}
+\pgfkeys{/tikz/input noinv down/.add code={}{\pgf@circuit@oa@iplusupfalse}}
```

Comment:
Why is this key `input noinv down` that instead of `noinv input down`?

Reply:
Oops. Well, easy to fix. Would you mind to open an issue? I will add the correct key form and leave  the wrong one hidden to avoid compatibility issues.

Reply:
Never mind, I did it --- see issue #315